app/forms/work_form.rb

```diff
@@ -135,9 +136,10 @@ def persisted?
 
   def content_file_presence
```

Comment:
Since this method is now handling not only presence of content files but the "too many" case, would you consider renaming this method?